packages/pn-commons/src/components/Data/ItemsTable.tsx

```diff
@@ -117,7 +115,7 @@ function ItemsTable<ColumnId extends string>({
                 aria-rowindex={index + 1}
               >
                 {columns.map((column) => {
-                  const cellValue = column.getCellLabel(row[column.id as keyof Notification], row);
+                  const cellValue = column.getCellLabel(row[column.id as keyof Item], row);
```

Comment:
@NicolaGiornetta since `ItemsTable` is a generic component which is indeed able to handle generic Items instead of just Notifications, I changed this declaration and got rid of the import of `Notifications`.